webapp/components/Group/GroupForm.vue

```diff
@@ -161,6 +172,7 @@ export default {
     return {
       categoriesActive: this.$env.CATEGORIES_ACTIVE,
       disabled: false,
+      descriptionMin: 50,
```

Comment:
There is a constant for it in `webapp/constants/groups.js`.

Reply:
donw